language/stdlib/modules/LibraAccount.move

```diff
@@ -914,8 +910,19 @@ module LibraAccount {
         make_account<Token>(new_account, auth_key_prefix, false)
     }
 
-    ///////////////////////////////////////////////////////////////////////////
-    // End of the proof of concept code
-    ///////////////////////////////////////////////////////////////////////////
+    public fun create_validator_operator_account<Token>(
+        creator_account: &signer,
```

Comment:
I believe only the association root can create a validator operator. If that's the case, could you have this take a `_: &Capability<AssociationRootRole>` (similar to `create_parent_vasp_account`)?